### ISSUE TITLE
Import POI's from GPX file

### ISSUE THREAD

**Issue by joshbowyer** [rinigus/pure-maps#553]
One of the most popular mobile map apps is Osmand. It stores POI's in GPX format. I believe it would be ideal to allow importing the favourites.gpx file that Osmand creates into PureMaps. I have done so manually for my own installation:

PureMaps syntax:

`    {
        "address": "123 Burger Way, Chicago, Illinois, USA",
        "bookmarked": true,
        "link": "",
        "phone": "",
        "poiId": "4a2ffecf-8d8f-4d90-ae9b-e10de7bdf71b",
        "poiType": "",
        "postcode": "",
        "provider": "",
        "shortlisted": false,
        "text": "",
        "title": "McDonald's",
        "type": "",
        "x": -91.000000,
        "y": 47.0000000
    }
`

Osmand syntax:

`<wpt lat="47.0000000" lon="-91.000000">
<name>McDonald's</name>
<type>Restaurants_Folder</type>
<extensions>
<osmand:icon>hamlet</osmand:icon>
<osmand:background>circle</osmand:background>
<osmand:color>#00842b</osmand:color>
</extensions>
</wpt>`

This is a simple example as I dont create POI's with all the data fields, but all that is needed is a basic field mapping for coordinates and the name.

**Comment by programmin1** [comment]
Where are the bookmark folders in Pure-maps, I didn't see the add/remove/list bookmarks feature while testing it out?
I too would like to import OsmAnd bookmarks to Linux phone.

**Comment by janjaapmeijer** [comment]
Ideally POI's locally stored in GPX format that can be synced to your mobile device with f.e. Syncthing.

**Comment by janjaapmeijer** [comment]
> Where are the bookmark folders in Pure-maps, I didn't see the add/remove/list bookmarks feature while testing it out? I too would like to import OsmAnd bookmarks to Linux phone.

POIs are stored in the folder: ~/.var/app/io.github.rinigus.PureMaps/config/io.github.rinigus.PureMaps/pois.json

**Comment by rinigus** [comment]
I suspect that there will be fields that are not compatible between Pure Maps and other maps apps. As other apps are evolving as well, even if we take someone's format over, it will be hard to follow it. Probably better is to have external conversion tools to switch between formats.
